add getOwnPropertyDescription Entry (#4092)
Codecademy/docs: content/javascript/concepts/objects/terms/getOwnPropertyDescriptors/getOwnPropertyDescriptors.md

---

> Title: '.getOwnPropertyDescriptors'

Reviewer: ```suggestion
Title: 'Object.getOwnPropertyDescriptors()'
```
Use this format whenever you write the name of the method.

> Description: 'returns all own property descriptors of a given object'

Reviewer: ```suggestion
Description: 'Returns all own property descriptors of a given object.'
```

Subjects:
  - 'Web Development'
  - 'Computer Science'
  - 'Web Applications'
Tags:
  - 'Properties'
  - 'Objects'
  - 'Functions'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/create-a-back-end-app-with-javascript'
---

> The **`Object.getOwnPropertyDescriptors()`** method returns all own property descriptors of a given object. It is useful for extracting property descriptors, including configurable, enumerable, value, and writable properties, from an object.manipulation.

Reviewer: ```suggestion
The **`Object.getOwnPropertyDescriptors()`** method returns all own property descriptors of a given object. It is useful for extracting property descriptors, including configurable, enumerable, value, and writable properties, allowing manipulation of these property attributes from an object.
```

## Syntax

```pseudo
Object.getOwnPropertyDescriptors(obj)
```

> `obj` is the given obeject.

Reviewer: ```suggestion
- `obj`: It is the given object.
```

> ## Examples

Reviewer: Remove this heading and add the rest of the two (`Example 1` & `Example 2`) headings as h2.

The following codes demonstrate a basic implementation of the `Object.getOwnPropertyDescriptors()` method:

### Example 1

> In this example, Object.getOwnPropertyDescriptors() returns an object where each key is a property name, and the corresponding value is the property descriptor for that property.

Reviewer: ```suggestion
In this example, `Object.getOwnPropertyDescriptors()` returns an object where each key is a property name, and the corresponding value is the property descriptor for that property.
```

```js
const obj = {
  name: 'John',
  age: 25,
};

const propertyDescriptors = Object.getOwnPropertyDescriptors(obj);

> 

Reviewer: ```suggestion
```
